### ISSUE TITLE
Cannot use `set()` with `.get()` and any other SetParams options

### ISSUE THREAD

**Issue by narzach** [redis/jedis#2340]
### Expected behavior

From the [docs](https://redis.io/commands/set), with Redis 6.2 I would expect to be able to use `set` with `nx()` and `get()` SetParams simultaneously (or any other SetParams options such as `keepttl()`). Instead, a syntax error is thrown.
```

String nullValue = jedis.set("key", "alpha", setParams().nx().get());
assertEquals(originalValue, null);
String previousValue = jedis.set("key", "beta", setParams().nx().get());
// because of .nx(), value should not have changed
assertEquals(previousValue, "alpha");
assertEquals(jedis.get("key"), "alpha");
```

### Actual behavior

the `set()` call fails with "ERR syntax error"

### Steps to reproduce:

Please create a reproducible case of your problem. Make sure
that case repeats consistently and it's not random
1. `jedis.set("key", "alpha", setParams().nx().get());`

#### Jedis version:

3.4.1

#### Redis version:

6.2

#### Java version:

8

**Comment by sazzad16** [comment]
@narzach How did you use Redis 6.2?

**Comment by narzach** [comment]
@sazzad16 There is a Docker image available on Docker Hub for Redis 6.2

https://hub.docker.com/_/redis

**Comment by sazzad16** [comment]
@narzach Redis 6.2 is yet to be released, including Docker. The available ones are RCs (release candidate).

From the errors message, it seems that the error is sent by Redis, not generated by Jedis. There is a possibility that `get` option under `set` command is not included in the Docker image.

I'm closing this issue. If you're sure that it is Jedis specific issue, by testing with other clients like redis-cli, feel free to reopen the issue.
